src/main/java/org/pac4j/undertow/DemoHandlers.java

```diff
@@ -138,8 +146,24 @@ public void handleRequest(HttpServerExchange exchange) throws Exception {
                 sb.append("<input type=\"submit\" name=\"submit\" value=\"Submit\" />");
                 sb.append("</form>");
 
-                exchange.getResponseSender().send(sb.toString());
+                returnHtml(exchange, sb.toString());
             }
         };
     }
+
```

Comment:
I am not sure but is it not implemented here:
https://github.com/pac4j/undertow-pac4j/blob/master/src/main/java/org/pac4j/undertow/ClientAuthenticationMechanism.java#L78
?


2015-04-30 8:45 GMT+02:00 LELEU Jérôme <notifications@github.com>:

> In src/main/java/org/pac4j/undertow/DemoHandlers.java
> <https://github.com/pac4j/undertow-pac4j-demo/pull/3#discussion_r29406761>
> :
>
> >              }
> >          };
> >      }
> > +
>
> In fact, it doesn't even work: I just kept the source code to remember.
>
> When trying to access a protected area, if you give up during the
> authentication process (like for Twitter), a 403 page is returned: it's a
> blank page here and I'd like to set my own 403 page...
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pac4j/undertow-pac4j-demo/pull/3/files#r29406761>.
>